### ISSUE TITLE
management/manifest endpoint broken in 9.0.1

### ISSUE THREAD

**Issue by thormick** [alerta/docker-alerta#442]
When running against Alerta 9.0.0 in Docker container then invoking the management/manifest results in a 500 server error:

```sh
$ curl -H "Authorization: Key $my_key_goes_here" http://localhost:8880/api/management/manifest
{"code": 500, "errors": ["Traceback (most recent call last):\n  File \"/venv/lib/python3.8/site-packages/flask/app.py\", line 1820, in full_dispatch_request\n    rv = self.dispatch_request()\n  File \"/venv/lib/python3.8/site-packages/flask/app.py\", line 1796, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)\n  File \"/venv/lib/python3.8/site-packages/flask_cors/decorator.py\", line 128, in wrapped_function\n    resp = make_response(f(*args, **kwargs))\n  File \"/venv/lib/python3.8/site-packages/alerta/auth/decorators.py\", line 53, in wrapped\n    return f(*args, **kwargs)\n  File \"/venv/lib/python3.8/site-packages/alerta/management/views.py\", line 74, in manifest\n    'build': build.BUILD_NUMBER,\nAttributeError: module 'alerta.build' has no attribute 'BUILD_NUMBER'\n"], "message": "module 'alerta.build' has no attribute 'BUILD_NUMBER'", "requestId": "5d076e00-1eae-45ea-8302-59d6e274efcb", "status": "error"}
```

This worked fine in the previous release; if I change the image for the container to 8.7.0 the same command runs without a hitch:

```sh
$ curl -H "Authorization: Key $my_key_goes_here" http://localhost:8880/api/management/manifest
{"build":"PROD","date":"2021-12-06T08:07:22Z","release":"8.7.0","revision":"5e6aa75b6fb7dab554d83f5efa7b505035ca26cd"}
```

I'm using a fairly standard docker-compose:
```yaml
  alerta-web:
    image: alerta/alerta-web:9.0.0
    # volumes:
      # - $PWD/config/config.json:/web/config.json
    ports:
      - 8880:8080
    depends_on:
      - alerta-db
    environment:
      # - DEBUG=1  # remove this line to turn DEBUG off
      - DATABASE_URL=postgres://postgres:postgres@alerta-db:5432/monitoring
      - AUTH_REQUIRED=True
      - ADMIN_USERS=admin@gapit.io,tms@gapit.io
      # - ADMIN_PASSWORD=super-secret # default is "alerta"
      # - ADMIN_KEY=admin-key  # assigned to first user in ADMIN_USERS
      # - ADMIN_KEY_MAXAGE=500
      # - PLUGINS=remote_ip,reject,heartbeat,blackout,normalise,enhance
    restart: unless-stopped
```

I see something similar has been an issue long in the past, but I can't see any obvious reason this would be related: #120

**Comment by thormick** [comment]
Didn't notice 9.0.1 was out until 60 seconds after hitting "Submit"; reproduced this there too.

**Comment by satterly** [comment]
Fixed in 9.0.2
